ci/test/06_script_b.sh

```diff
@@ -8,6 +8,9 @@ export LC_ALL=C.UTF-8
 
 cd "build/bitcoin-$HOST" || (echo "could not enter distdir build/bitcoin-$HOST"; exit 1)
 
+mkdir -p contrib
+cp -R ../../contrib/ contrib/
```

Comment:
I think you can't run these with the `travis` user, only with the docker `root` user.

I.e. 
```suggestion
DOCKER_EXEC cp -R ../../contrib/ contrib/
```